src/components/elements/Unavatar/Unavatar.js

```diff
@@ -21,8 +21,8 @@ const Unavatar = ({ query, ...props }) => {
   )
 }
 
-const UnavatarMicrolink = props => (
-  <StyledImage alt={`Logo for ${props.url}`} {...props} />
+const UnavatarMicrolink = ({ url, ...props }) => (
+  <StyledImage alt={`Logo for ${url}`} {...props} />
```

Comment:
in this way, it's not passing `props.url` down, is that expected?

Reply:
Yep that's correct, otherwise the `url` prop was being passed to the `<img>` element - causing a warning about an invalid prop passed to a DOM element.